### DISCUSSION TITLE
Creating resources.arsc programatically

### DISCUSSION THREAD

**Discussion by zt64** [REAndroid/ARSCLib#42] [General]
How do I use this to create resources.arsc entirely through code? I'll mainly be using png's and some files in the values resource directory.

**Comment by REAndroid** [comment]
You can also create complete apk check example on [REAME.md](https://github.com/REAndroid/ARSCLib#examples)

**Reply by zt64** [reply]
Perfect. Do you plan to add more documentation and examples? I wanted to add a different app icon but I'm not sure how
